packages/l/lsscsi/package.yml

```diff
@@ -0,0 +1,17 @@
+name       : lsscsi
+version    : 0.32
+release    : 1
+source     :
+    - http://sg.danny.cz/scsi/lsscsi-0.32.tgz : 0a800e9e94dca2ab702d65d72777ae8cae078e3d74d0bcbed64ba0849e8029a1
+homepage   : http://sg.danny.cz/scsi/lsscsi.html
+license    : GPL-2.0-or-later
+component  : system.base
```

Comment:
Let's change this to `system.utils`. Packages in `system.base` are special, and are always installed on systems. I don't think we want that here.